packages/koenig-lexical/src/plugins/EmojiPickerPlugin.jsx

```diff
@@ -42,7 +42,14 @@ export function EmojiPickerPlugin() {
         }
 
         async function searchEmojis() {
-            const filteredEmojis = await SearchIndex.search(queryString);
+            let filteredEmojis = [];
+            if ([')','-)'].includes(queryString)) {
+                filteredEmojis = await SearchIndex.search('smile');
+            } else if (['(',':-('].includes(queryString)) {
```

Comment:
Do we need to remove the leading `:` in `:-(` here?

Reply:
That's what I get for letting copilot autofill that line without double checking it 👎